tests/e2e/state.go

```diff
@@ -492,27 +495,31 @@ type ValPubKey struct {
 }
 
 func (tr TestRun) getValPower(chain chainID, validator validatorID) uint {
+	if *verbose {
+		log.Println("getting validator power for chain: ", chain, " validator: ", validator)
+	}
 	//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
-	bz, err := exec.Command("docker", "exec", tr.containerConfig.instanceName, tr.chainConfigs[chain].binaryName,
+	command := exec.Command("docker", "exec", tr.containerConfig.instanceName, tr.chainConfigs[chain].binaryName,
 
 		"query", "tendermint-validator-set",
 
 		`--node`, tr.getQueryNode(chain),
-	).CombinedOutput()
+	)
+	bz, err := command.CombinedOutput()
 	if err != nil {
-		log.Fatalf("error: %v", err)
+		log.Fatalf("encountered an error when executing command '%s': %v, output: %s", command.String(), err, string(bz))
 	}
 
 	valset := TmValidatorSetYaml{}
 
 	err = yaml.Unmarshal(bz, &valset)
 	if err != nil {
-		log.Fatalf("error: %v", err)
+		log.Fatalf("yaml.Unmarshal returned an error while unmarshalling validator set: %v, input: %s", err, string(bz))
```

Comment:
nit: would it make sense to print the validator set here as well (similarly below).

Reply:
The validator set (valSet) is probably garbage since the unmarshalling failed, right? Or do you mean a different variiable?